### PR TITLE
Adds a new skrell subspecies: Axiori

### PR DESCRIPTION
**Pull request by Casper3667**
Adds a new skrell subspecies. Mechanically they
- take slightly more burn damage
- take slightly less brute & oxy damage
- runs slightly slower than other skrell (but faster than humans)
- get thirsty faster

image
image
image
image

### CODE REVIEW

**Comment by Casper3667** [comment]
!review

File: code/game/objects/items/devices/auto_cpr.dm [hunk 1]
```diff
@@ -12,7 +12,7 @@
 	. = ..()
 	if(force || !istype(H) || slot != slot_wear_suit)
 		return
-	if(H.species.get_bodytype() in list(SPECIES_HUMAN, SPECIES_HUMAN_OFFWORLD, SPECIES_TAJARA, SPECIES_TAJARA_ZHAN, SPECIES_TAJARA_MSAI, SPECIES_SKRELL, SPECIES_UNATHI)) //gtfo stinky bugs
+	if(H.species.get_bodytype() in list(SPECIES_HUMAN, SPECIES_HUMAN_OFFWORLD, SPECIES_TAJARA, SPECIES_TAJARA_ZHAN, SPECIES_TAJARA_MSAI, SPECIES_SKRELL,SPECIES_SKRELL_AXIORI, SPECIES_UNATHI)) //gtfo stinky bugs
```
Comment (NonQueueingMatt) [inline code/game/objects/items/devices/auto_cpr.dm:15]: the literal fuck is this

**Review by NonQueueingMatt — APPROVED** [review]
Mechanically, this looks fine.

File: code/game/objects/items/devices/auto_cpr.dm (hunk 1 above)
Comment (Casper3667) [reply code/game/objects/items/devices/auto_cpr.dm:15]: I have no idea.

Comment (Alberyk) [reply code/game/objects/items/devices/auto_cpr.dm:15]: This is probably broken, since bodytypes are different from species names.

**Review by Arrow768 — APPROVED** [review]

**Pull request merged**